syntax = "proto3";

package test.integration.filters;

import "validate/validate.proto";

message LoggingTestFilterConfig {
  string logging_id = 1 [(validate.rules).string = {min_len: 1}];
  string expected_cluster_name = 2 [(validate.rules).string = {min_len: 1}];
  bool expect_stats = 3;
  bool expect_envoy_grpc_specific_stats = 4;
  bool expect_response_bytes = 5;
}
